### PR TITLE
Update spoiler nightmode

### PR DESCRIPTION
**Pull request by roshkins**
Now it's orange to correspond to candy stripes!

### CODE REVIEW

**Comment by andytuba** [comment]
ref https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3899

File: lib/css/modules/_neverEndingReddit.scss [hunk 1]
```diff
@@ -115,13 +115,18 @@
...
+	}
+
 	&::after {
 		font: normal 100% Batch;
 		color: #888;
 		display: flex;
 		align-items: center;
 		width: 10px;
 		height: 10px;
+		padding: 0 2px 4px;
```
Comment (larsjohnsen) [inline lib/css/modules/_neverEndingReddit.scss]: Padding is specified below, in
```	&::after,
	&.paused.reversePause::after {
		content: '\F16C';
		padding: 2px;
	}

	&.paused::after,
	&.reversePause::after {
		content: '\F16B';
		padding: 2px 1px 2px 3px;
	}
```

**Comment by erikdesjardins** [comment]
Thanks!

**Comment by roshkins** [comment]
woo!

**Pull request merged**